model-api-gen-gradle-test/vue-integration/build.gradle.kts

```diff
@@ -5,6 +5,25 @@ plugins {
     alias(libs.plugins.node)
 }
 
+/**
+ * This task is needed because the listed packages have been rebuilt (locally or CI).
+ * As a result, the verification hashes in package-lock.json are out of date.
+ * Reinstalling them will update their hashes.
+ * This issue only arises because we are using the packaged libraries.
+ *
+ * We use packaged libraries for the following reason:
```

Comment:
I suggest moving this below the paragraph starting at L16 and mark it with (b) so it's clearer, that we use b out of those two options.

Reply:
Done